src/main/java/Group15/View/WorkoutView.java

```diff
@@ -50,9 +50,12 @@ public static Scene createScene(Workout workout)
     private static Pane createTitlePane()
     {
         HBox titlePane = new HBox();
-        titlePane.setAlignment(Pos.TOP_CENTER);
+        titlePane.setAlignment(Pos.CENTER_LEFT);
```

Comment:
Because you are alligning CENTER_LEFT, the title gets shifted to the left. Did you not test the changes you made?